### ISSUE TITLE
Documenting an array of files

### ISSUE THREAD

**Issue by Nordok** [knuckleswtf/scribe#128]
How can I document an array of files? If I just write `file[]` it breaks request example (empty body)

<img width="1545" alt="Screenshot 2020-10-30 at 17 42 25" src="https://user-images.githubusercontent.com/13319258/97726402-d8a64c00-1ad7-11eb-8943-18bb8f963018.png">

Controller method:
```
    /**
     * Create
     * @bodyParam content string required Example: This is my first post. Hello @sam.smith. #hashtag#hashTAG
     * @bodyParam attachments file[]
     * @param CreatePostRequest $request
     * @return PostResource
     */
    public function store(CreatePostRequest $request)
    {
        ...
    }
```

Validation rules from CreatePostRequest class:
```
    public function rules()
    {
        return [
            'content' => ['required', 'string'],
            'attachments' => ['array', 'max:4'],
            'attachments.*' => ['image', 'max:1024']
        ];
    }
```

**Comment by Nordok** [comment]
@shalvah Are there any updates, by any chance?

**Comment by shalvah** [comment]
This should be fixed in 2.2.1.
